pkg/forwarder/backoff_test.go

```diff
@@ -0,0 +1,66 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package forwarder
+
+import (
+	"math"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+func TestMinBackoffFactorValid(t *testing.T) {
+	assert.True(t, minBackoffFactor >= 2)
+}
+
+func TestRandomBetween(t *testing.T) {
+	getRandomMinMax := func() (float64, float64) {
+		a := float64(rand.Intn(10))
+		b := float64(rand.Intn(10))
+		min := math.Min(a, b)
+		max := math.Max(a, b)
+		return min, max
+	}
+
+	for i := 1; i < 100; i++ {
+		min, max := getRandomMinMax()
+		between := randomBetween(min, max)
+
+		assert.True(t, min <= between)
+		assert.True(t, max >= between)
+	}
+}
+
+func TestGetBackoffDuration(t *testing.T) {
+	previousBackoffDuration := time.Duration(0) * time.Second
```

Comment:
Not sure what we really test here, can you add a comment ?

Also a couple of things that's be nice to test:
- We never go above `maxBackoffTime`
- Based on your comments, you know what will `GetBackoffDuration(0)` & `GetBackoffDuration(1)` so we should test those